packages/neo3-parser/README.MD

```diff
@@ -10,6 +10,107 @@
 </p>
 
 # Neo3-parser
+
 Neo3-parser is a specification of how SmartContract client SDKs can interact with different parsing libraries such as Neon-JS.
 
 Visit the [main page](../../README.md) of the project.
+
+## How to parse responses
+
+After invoking a contract you'll get the results on a stack. Use the `parseRpcResponse(field: RpcResponse, parseConfig?: ParseConfig)` function on each item to get the results parsed.
+
+### How to use parseConfig
+
+The `parseConfig` object has the following properties:
+
+| Property                  | Description                                                                               |
+|---------------------------|-------------------------------------------------------------------------------------------|
+| type: string              | a [valid ABI Type](https://github.com/neo-project/proposals/blob/master/nep-14.mediawiki#method)|
+| hint?: string             | a type that extends from the ABI type                                                     |
+| generic?: ParseConfig     | you only need to pass this prop if `type` is `"Array"`                                    |
+| genericKey?: ParseConfig  | you only need to pass this prop if `type` is `"Map"`                                      |
+| genericItem?: ParseConfig | you only need to pass this prop if `type` is `"Map"`                                      |
+| union?: ParseConfig[]     | you only need to pass this prop if `type` is `"Any"` and you expect to get multiple types |
+
+> Note: check [HINT_TYPES](./src/index.ts) to see what types are available to use on `hint`.
+
+### Example
+
+```ts
+// Simulating a result stack after invoking a contract
+const stackResult = [ 
+  {
+    type: "ByteString",
+    value: "AAECAwQFBgcICQoLDA0ODxAREhM=" // This value is a Hash160
+  } 
+]
+
+// Call parseRpcResponse on an item of the stack
+const response = Neo3Parser.parseRpcResponde(stackResult[0])
```

Comment:
you put `parseRpcResponde` with the D instead of S by mistake in several places 😅